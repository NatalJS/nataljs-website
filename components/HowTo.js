import React from 'react';

const HowTo = () => (
  <section id="how-to" className="section how-to">
    <h1 className="section-title">Por onde começar?</h1>
    <article className="article col-md-12 col-lg-8">
      <p className="article__body">
        Acompanhe nosso site, github e páginas do Facebook e Twitter (em breve) para ficar ligado
        nos eventos e projetos que estamos organizando. Como comunidade flat, todos podem contribuir
        igualmente e se tornar membros do projeto.
      </p>
      <footer className="text-center">
        <a href="https://github.com/NatalJS" className="btn--negative">GITHUB</a>
      </footer>
    </article>
  </section>
);

export default HowTo;
