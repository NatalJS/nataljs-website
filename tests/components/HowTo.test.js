
import renderer from 'react-test-renderer';
import HowTo from 'components/HowTo';

describe('Component HowTo', () => {
  let Component;

  beforeAll(() => {
    Component = setup(HowTo);
  });

  it('should render component correctly', () => {
    const wrapper = renderer.create(Component());

    expect(wrapper.toJSON()).toMatchSnapshot();
  });
});
