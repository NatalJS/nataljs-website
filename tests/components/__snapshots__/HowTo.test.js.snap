// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component HowTo should render component correctly 1`] = `
<section
  className="section how-to"
  id="how-to"
>
  <h1
    className="section-title"
  >
    Por onde começar?
  </h1>
  <article
    className="article col-md-12 col-lg-8"
  >
    <p
      className="article__body"
    >
      Acompanhe nosso site, github e páginas do Facebook e Twitter (em breve) para ficar ligado nos eventos e projetos que estamos organizando. Como comunidade flat, todos podem contribuir igualmente e se tornar membros do projeto.
    </p>
    <footer
      className="text-center"
    >
      <a
        className="btn--negative"
        href="https://github.com/NatalJS"
      >
        GITHUB
      </a>
    </footer>
  </article>
</section>
`;
